src/main/java/com/rakovets/course/java/core/practice/oop_classes_and_objects/Atm.java

```diff
@@ -0,0 +1,37 @@
+package com.rakovets.course.java.core.practice.oop_classes_and_objects;
+
+public class Atm {
+    private int numberBanknotes20;
+    private int numberBanknotes50;
+    private int numberBanknotes100;
+
+    public Atm(int numberBanknotes20, int numberBanknotes50, int numberBanknotes100) {
+        this.numberBanknotes20 = numberBanknotes20;
+        this.numberBanknotes50 = numberBanknotes50;
+        this.numberBanknotes100 = numberBanknotes100;
+    }
+
+    public void addBanknotes20(int number) {
+        this.numberBanknotes20 += number;
+    }
+
+    public int getNumberBanknotes20() {
+        return numberBanknotes20;
+    }
+
+    public void addBanknotes50(int number) {
+        this.numberBanknotes50 += number;
+    }
+
+    public int getNumberBanknotes50() {
+        return numberBanknotes50;
+    }
```

Comment:
"Violation: Getters and setters should be placed at the very end
Solution: Move getters and setters to the end"